node/_fs/_fs_access.ts

```diff
@@ -57,7 +58,7 @@ export function access(
       );
       e.path = path;
       e.syscall = "access";
-      e.errno = os.errno.ENOENT;
+      e.errno = codeMap.get("EACCES");
```

Comment:
Should this be `codeMap.get("ENOENT")`?

Reply:
My bad

Reply:
Kinda worried that the current tests didn't catch this though